e2e/support/helpers/e2e-action-helpers.js

```diff
@@ -9,8 +9,10 @@ export function setActionsEnabledForDB(dbId, enabled = true) {
 }
 
 export function fillActionQuery(query) {
-  cy.get(".ace_content:visible").type(query, {
+  // Without this wait, content tends to drop from the beginning of the string. TODO: Fix
+  cy.get(".ace_content:visible").wait(500).type(query, {
```

Comment:
We use `.focus().clear()` to fix most flakes with the ace editor https://github.com/metabase/metabase/blob/4be9fcf47fd64a6e30938b24ca5d0cff1b2d91a4/e2e/support/helpers/e2e-custom-column-helpers.js#L10. Could it work here as well?

Reply:
I'll give it a try!

Reply:
Unfortunately, the change wasn't super stable. I'm going to make a note in the epic to come back and improve this though